.github/workflows/other-arch.yml

```
@@ -0,0 +1,75 @@
+name: Other-architectures
+
+# https://www.jeffgeerling.com/blog/2020/running-github-actions-workflow-on-schedule-and-other-events
+# https://github.com/uraimo/run-on-arch-action
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 2 * * SUN'
+
+jobs:
+  build-other-architectures:
+    # The host should always be linux
+    runs-on: ubuntu-20.04
+    name: Build on ${{ matrix.distro }} ${{ matrix.arch }}
+
+    # Run steps on a matrix of 3 arch/distro combinations
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        #  - arch: armv6
+        #    distro: buster #bullseye
+        #    target: ARMV6
+        #  - arch: armv7
+        #    distro: ubuntu20.04
+        #    target: ARMV7
+          - arch: aarch64
+            distro: ubuntu20.04 #fedora_latest
+            target: ARMV8
+          - arch: ppc64le
+            distro: ubuntu20.04
+            target: POWER8
+          - arch: s390x
+            distro: ubuntu20.04
+            target: Z13
+
+    steps:
+    # https://github.com/marketplace/actions/cancel-workflow-action
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
+      with:
+        access_token: ${{ github.token }}
+
+    - name: Checkout repository
+      uses: actions/checkout@v2.1.0
```

Why an old release?  https://github.com/actions/checkout/releases
```suggestion
      uses: actions/checkout@v2
```